contrib/pyln-proto/pyln/proto/message/array_types.py

```diff
@@ -8,11 +12,11 @@ class ArrayType(FieldType):
 wants an array of some type.
 
     """
-    def __init__(self, outer, name, elemtype):
+    def __init__(self, outer: 'SubtypeType', name: str, elemtype: FieldType):
```

Comment:
Seems you were far quicker than me finding out about recursive type definitions being possible via string annotations :wink:

I got quite lost when trying to figure out how to annotate something with a type that'd be defined later...